### DISCUSSION TITLE
Mcserver wont start

### DISCUSSION THREAD

**Discussion by nightdancerpl** [itzg/docker-minecraft-server#1959] [Q&A]
Hi last time my openmediavault server crashed and i need to reinstall docker, now when i try to run minecraft it looks like that

`$ docker run -d -it -e EULA=TRUE -e VERSION=LATEST -e MEMORY=2G -e ANNOUNCE_PLAYER_ACHIEVEMENTS=true -p 25565:25565 -v /srv/dev-disk-by-label-1/Files/Minecraft:/data --name Minecraft itzg/minecraft-server
Unable to find image 'itzg/minecraft-server:latest' locally
latest: Pulling from itzg/minecraft-server
7608715873ec: Pull complete
64a0b7566174: Pull complete
342199186d11: Pull complete
e3f00422fe3f: Pull complete
be46c779d08c: Pull complete
88cb54e68cfd: Pull complete
c24cdab986de: Pull complete
26d700f0a1c5: Pull complete
639dd8b38267: Pull complete
261d7d8181f0: Pull complete
bd82a0628f20: Pull complete
9b003fa03cbe: Pull complete
b0e87c87ed37: Pull complete
891a0e00d492: Pull complete
df53ba62eb12: Pull complete
4f4fb700ef54: Pull complete
197ac3c5f118: Pull complete
30ac009f7b44: Pull complete
49995c6a5a91: Pull complete
98617e0f3cdc: Pull complete
293f1407e5fc: Pull complete
a6a9f6d7c654: Pull complete
94881da3a9b1: Pull complete
Digest: sha256:00d24cf7e2663f793880ed60ced2da188dea476c330600506fe42eabf7ceaac1
Status: Downloaded newer image for itzg/minecraft-server:latest
aaab4f6e5a020f7f95a349c22ba55ef259916d021b7dcf71f4ecd0b72ab9d5a9
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: unable to apply apparmor profile: apparmor failed to apply profile: write /proc/self/attr/apparmor/exec: no such file or directory: unknown.`

**Comment by itzg** [comment]
It looks like apparmor is "getting in the way". Hopefully searching around for "apparmor docker" will reveal some solutions. Not ideal, but disabling apparmor entirely would probably fix it.
